al_aws_collector.js

```diff
@@ -700,11 +714,20 @@ class AlAwsCollector {
         
         if (messages && messages.length > 0) {
             m_alCollector.AlLog.buildPayload(
-                    collector._collectorId, collector._collectorId, hostmetaElems, messages, formatFun, function(err, payload){
+                    collector._collectorId, collector._collectorId, hostmetaElems, messages, formatFun, function(err, payloadObj){
                 if (err) {
                     return callback(err);
                 } else {
-                    return collector.send(payload, false, ingestType,  callback);
+                    // send the lmc stats if ingest type is logmsgs
+                    async.waterfall([
+                        function (asyncCallback) {
+                            return collector.send(payloadObj.payload, false, AlAwsCollector.IngestTypes.LOGMSGS, asyncCallback);
+                        },
+                        function (asyncCallback) {
+                            const stats = collector.prepareLmcStats(payloadObj.raw_count, payloadObj.raw_bytes);
```

Comment:
Do we have unit tests covering these changes?

Reply:
I have added the test case for [send lmc stats](https://github.com/alertlogic/al-aws-collector-js/blob/master/test/al_aws_collector_test.js#L627) method